client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java

```diff
@@ -509,6 +519,19 @@ protected String getPath(Method method, Object[] arguments, JaxRsMeta meta) {
                     } else {
                         path = path.replaceAll("\\{" + ((PathParam)annotation).value() + "\\}", this.urlEncode(this.serialize(value)));
                     }
+                } else if (annotation instanceof BeanParam) {
+                    if (value == null) {
+                        continue;
+                    }
+                    beanParamCache
+                        .computeIfAbsent(types[i], this::getBeanParamGetters)
+                        .forEach(v -> {
+                            Function<Object, Object> beanPropertyGetter = v.getValue1();
+                            Object beanPropertyValue = beanPropertyGetter.apply(value);
+                            Annotation[] annotations = v.getValue2();
```

Comment:
Again regarding the Tuple, it makes this code very hard to follow. It also requires the developer to keep track of which value is saved in which of the values.

Reply:
Disregarding this piece of code, should we add the Tuple class to utils or not? I had the need to simply return two values without any fuzz, and I've had this problem many times before, so this time I added the Tuple, thinking it would be useful in more places. Anyone else who has opinions about the Tuple concept?